Makefile.build

```diff
@@ -224,6 +236,7 @@ build-manifests-operator: "$(GOBIN)/addlicense" "$(BIN_DIR)/kustomize" $(OUTPUT_
 	@ echo "    $(ADMISSION_WEBHOOK_IMAGE): $(ADMISSION_WEBHOOK_TAG)"
 	@ echo "    $(OCI_SYNC_IMAGE): $(OCI_SYNC_TAG)"
 	@ echo "    $(HELM_SYNC_IMAGE): $(HELM_SYNC_TAG)"
+	@ echo "    $(ASKPASS_IMAGE): $(ASKPASS_TAG)"
```

Comment:
Need to add `-e "s|ASKPASS_IMAGE_NAME|$(ASKPASS_TAG)|g" \` to the `sed` call below here

Reply:
good catch.  fixed.